packages/block-library/src/query-pagination-next/edit.js

```diff
@@ -1,22 +1,78 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
-import { useBlockProps, PlainText } from '@wordpress/block-editor';
+import { __, _x } from '@wordpress/i18n';
+import {
+	useBlockProps,
+	PlainText,
+	InspectorControls,
+} from '@wordpress/block-editor';
+import {
+	PanelBody,
+	__experimentalSegmentedControl as SegmentedControl,
+	__experimentalSegmentedControlOption as SegmentedControlOption,
+} from '@wordpress/components';
 
 export default function QueryPaginationNextEdit( {
-	attributes: { label },
+	attributes: { label, arrow },
 	setAttributes,
 } ) {
 	return (
-		<PlainText
-			__experimentalVersion={ 2 }
-			tagName="a"
-			aria-label={ __( 'Next page link' ) }
-			placeholder={ __( 'Next Page' ) }
-			value={ label }
-			onChange={ ( newLabel ) => setAttributes( { label: newLabel } ) }
-			{ ...useBlockProps() }
-		/>
+		<div className="wp-block-query-pagination-next">
```

Comment:
We shouldn't add extra markup here. The block classes are added through `useBlockProps`. We can use a `fragment(<>)` here.